assets/js/blocks/mini-cart/block.tsx

```diff
@@ -34,6 +34,7 @@ import QuantityBadge from './quantity-badge';
 import { MiniCartContentsBlock } from './mini-cart-contents/block';
 import './style.scss';
 import { blockName } from './mini-cart-contents/attributes';
+import { useTypographyProps } from '../../hooks/style-attributes';
```

Comment:
> So, perhaps, we could move this file into the base/hooks directory and change at least this import to use the alias?

@sunyatasattva I'm in favor of this! I think that will add some consistency and the more we do that, the easier our codebase becomes to reason about overall.

That being said, because this relative path is used across a handful of other files that are unrelated to this PR, perhaps it would be better to circle back and target them all in one go in a separate PR. Does that sound good?

I'm happy to handle that once I merge this in.

Reply:
I've added a new Issue to address this task at #6866.